### PR TITLE
Support packaging as .whl files

### PR DESCRIPTION
**Pull request by derekbekoe**
When publishing on PyPI, we should publish both .tar.gz and .whl files.

Add setup.cfg files
- Update package-verify and nightly-build to use bdist_wheel also

### CODE REVIEW

File: scripts/command_modules/package_verify.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def build_package(path_to_package, dist_dir):
     print_heading('Building {}'.format(path_to_package))
     path_to_setup = os.path.join(path_to_package, 'setup.py')
     set_version(path_to_setup)
-    cmd_success = exec_command('python setup.py sdist -d {}'.format(dist_dir), cwd=path_to_package)
+    cmd_success = exec_command('python setup.py sdist bdist_wheel -d {}'.format(dist_dir), cwd=path_to_package)
```
Comment (yugangw-msft) [inline scripts/command_modules/package_verify.py]: Just a question, is it worthwhile to produce both packages?

**Comment by yugangw-msft** [comment]
LGTM

File: scripts/command_modules/package_verify.py (hunk 1 above)
Comment (derekbekoe) [inline scripts/command_modules/package_verify.py]: Yes they recommend creating both and uploading both.

**Comment by derekbekoe** [comment]
I've verifying a bit more the change before merging.

**Pull request merged**